src/utils/typescript.ts

```diff
@@ -22,6 +22,7 @@ export const tryToRegisterTsNode = (): void => {
 
         if (JSON.parse(swcRaw)) {
             try {
+                require("@swc/core");
```

Comment:
Require @swc/core to get an error with code "MODULE_NOT_FOUND" if it is not installed

Reply:
This way `testplane: you may install @swc/core for significantly faster reading of typescript tests.` will be shown